project/database.py

```diff
@@ -2,22 +2,24 @@
 from werkzeug.security import generate_password_hash
 from project.config import conn
 
+cur = conn.cursor()
 
 class Request():
     """requests helper"""
 
-    def __init__(self):
+    def __init__(self, title, description, request_type):
 
-        self.conn = conn
-        self.cur = self.conn.cursor()
+        self. title = title
+        self.description = description
+        self.request_type = request_type
 
     def request_exists(self, user_id, title):
         """
         Check if a user exists
         """
-        self.cur.execute(
+        cur.execute(
             "SELECT * FROM requests WHERE user_id = %s and title = %s ;", (user_id, title))
-        self.request = self.cur.fetchone()
+        self.request = cur.fetchone()
 
     def create_request(self, title, description, request_type, date_created, user_id):
```

Comment:
Since you've initialised title, description and request_type, you don't need to have them as variables of this method again. You can just call them in your methods for example for title `self.title`